glide.yaml

```diff
@@ -0,0 +1,31 @@
+package: github.com/projectriff/function-sidecar
+import:
+- package: github.com/Shopify/sarama
+  version: ~1.15.0
+- package: github.com/bsm/sarama-cluster
+  version: ~2.1.12
+- package: github.com/giantswarm/retry-go
+  version: ~0.5.0
+- package: github.com/golang/protobuf
+  version: ~1.0.0
+  subpackages:
+  - proto
+- package: github.com/projectriff/message-transport
+  subpackages:
```

Comment:
Should make `version: master` explicit here, for consistency with other projects